package.json

```diff
@@ -101,9 +102,8 @@
     "eslint-plugin-react-hooks": "~4.1.0",
     "history": "4.10.1",
     "husky": "~4.3.0",
-    "jest": "24.9.0",
```

Comment:
@JacobMGEvans why removing jest plz?

Reply:
React Scripts has Jest under the hood. The tests are still running

Reply:
If we still need it we can put it back just seems redundant